src/Symfony/Component/Asset/EventListener/PreloadListener.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Asset\EventListener;
+
+use Symfony\Component\Asset\Preload\HttpFoundationPreloadManager;
+use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
+
+/**
+ * Adds preload's Link HTTP headers to the response.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+class PreloadListener
```

Comment:
should be an event subscriber IMO, like our other listeners, to embed the knownledge about the event being listened to